internal/fakechain/fakechain.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/core/interop"
 	"github.com/nspcc-dev/neo-go/pkg/core/mempool"
 	"github.com/nspcc-dev/neo-go/pkg/core/native"
+	"github.com/nspcc-dev/neo-go/pkg/core/nep17"
```

Comment:
That's probably more of RPC data then, `core` has nothing to do with that (and the balance getting logic can be moved there).

Reply:
If we're to move `GetNEP17Balances` to RPC, then we need to export management's `GetNEP17Contracts` as far as DAO's `GetNEP17TransferInfo`, I'm not sure that it's what we want.

But I agree that `core` is not the proper place for nep17 `Balances` structure, so perhaps we may think about some other place for it? (RPC is not suitable, because the import cycle happens then).

Reply:
Fixed by exposing `GetNEP17Contracts` and `GetNEP17LastUpdated`.